source/dedup_source.go

```diff
@@ -43,7 +43,7 @@ func (ms *dedupSource) Endpoints() ([]*endpoint.Endpoint, error) {
 	}
 
 	for _, ep := range endpoints {
-		identifier := ep.DNSName + " / " + ep.Target
+		identifier := ep.DNSName + " / " + ep.Targets[0]
```

Comment:
why only first target is considered ?

Reply:
We only aggregate the targets in the plan, so at this point we would only have 1 target